cdbot/constants.py

```diff
@@ -143,6 +144,7 @@ class Exchange:
 }
 
 # Admin Constants
+BANNED_WORDS = []
```

Comment:
There's already a banned words constant defined at https://github.com/CyberDiscovery/cyberdisc-bot/blob/dc4f20a048c4cfe528eddebf6054bb6043de3f39/cdbot/constants.py#L151. It would probably be better to pull this out into its own constant and use it, rather than having two copies of banned words.